src/hipscat/catalog/pixel_tree.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import annotations
+
+import pandas as pd
+
+from hipscat.catalog import PixelNode, PixelNodeType
+
+
+class PixelTree:
+    """Sparse Quadtree of HEALPix pixels that make up the HiPSCat catalog
+
+    This class store each node in the tree, with leaf nodes corresponding to pixels with data files.
+
+    There are a number of methods in this class which allow for quickly navigating through the
+    tree and performing operations to filter the pixels in the catalog.
+
+    Attributes: pixels: Nested dictionary of pixel nodes stored in the tree. Indexed by HEALPix
```

Comment:
I think you'll want each attribute on its own line to get picked up by pydoc.

Reply:
Fixed now, thanks